ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/PageType.kt

```diff
@@ -0,0 +1,61 @@
+package com.braintreepayments.api.shopperinsights
+
+/**
+ * The page or view that a button is displayed on.
+ */
+enum class PageType {
+    /**
+     * The homepage
+     */
+    HOMEPAGE,
```

Comment:
Does capitalization here matter? On iOS we are sending these all lowercase values - wondering if we should do the same on Android